client/src/App.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { QueryClient, QueryClientProvider } from 'react-query';
 
-import Header from '@/components/Header';
+import Header from '@/containers/HeaderContainer';
```

Comment:
```suggestion
import HeaderContainer from '@/containers/HeaderContainer';
```
혼동이 올 수 있으니 `HeaderContainer`로 호출하는 건 어떨까요?

Reply:
변경했습니다 :)